crates/project/src/lsp_store.rs

```diff
@@ -4906,96 +4907,97 @@ impl LspStore {
         let buffer = buffer.read(cx);
         let file = File::from_dyn(buffer.file())?;
         let abs_path = file.as_local()?.abs_path(cx);
-        let uri = lsp::Url::from_file_path(abs_path).unwrap();
-        let next_snapshot = buffer.text_snapshot();
+        if let Ok(uri) = lsp::Url::from_file_path(abs_path) {
```

Comment:
How are these changes in the `LspStore` related to the Vim mode indicator?

Reply:
Not directly, actually. This fixed failing unit tests on Windows. Should I pull this out into a separate PR?

Reply:
> Should I pull this out into a separate PR?

That would be great, thank you!

Reply:
Done :)